superset-frontend/src/components/ListView/types.ts

```diff
@@ -125,4 +103,5 @@ export enum FilterOperators {
   between = 'between',
   dashboardIsFav = 'dashboard_is_favorite',
   chartIsFav = 'chart_is_favorite',
+  status = 'status',
```

Comment:
is this being used in the current implementation?

Reply:
It is still being used here: https://github.com/apache/superset/blob/aaf88e2cd5b447c1412dd30f8db18e6b7a45e5b6/superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

Just wanna keep a trace here to make future migration more obvious...

Reply:
I might be missing something, but I don't see any reference to `FilterOperator.status` in that file, or any other file in the diff. 

Is the idea to add a status to other objects, charts, queries, datasets, etc? If no, it might make sense to prefix this filterOperator with `dashboard_` so it doesn't accidentally get used somewhere it's not implemented

Reply:
Oh, sorry, this indeed needs to be cleaned up. I was thinking the `status` column, not the operator.